### DISCUSSION TITLE
How to recover my account after deleting?

### DISCUSSION THREAD

**Discussion by leroyanders** [npm/feedback#755] [General]
Hey, my account(**nesspkg**) was deleted yesterday, I need to restore it, it's very important to me, there are packages of my new project, I can pay for premium, **it's really very important to me!!!**

**Comment by leroyanders** [comment]
Guys, it's really important!

**Reply by leroyanders** [reply]
Guys?

**Reply by ljharb** [reply]
File an npm support ticket; they won't likely be able to help you any other way.

**Reply by leroyanders** [reply]
> File an npm support ticket; they won't likely be able to help you any other way.

i email them, they didn't answer me

**Reply by ljharb** [reply]
when did you email them? emailing them on a weekend and expecting an answer by monday morning is a bit unreasonable.

**Reply by leroyanders** [reply]
> when did you email them? emailing them on a weekend and expecting an answer by monday morning is a bit unreasonable.

yesterday, today, just now

**Reply by ljharb** [reply]
then yes, be patient, today's the first day of the work week and they have the whole weekend to catch up on.

**Reply by leroyanders** [reply]
> then yes, be patient, today's the first day of the work week and they have the whole weekend to catch up on.

you're right, thank you!
